### ISSUE TITLE
Releases and Links out of Date

### ISSUE THREAD

**Issue by DrMerlin** [rdp/sensible-cinema#23]
Hey man- checking in with your project after a long time.  It looks like you've been busy making updates, but the releases are either impossible to download, horribly out of date, or both.

**Comment by misbach** [comment]
Yes, we need to spark some life into this project.

**Comment by rdp** [comment]
Yes it is out of date, though in reality I just have pivoted to the "play
it my way" online only version, and it gets updates like constantly.  But
the DVD player does need a new release if/when I circle back to it...

**Comment by misbach** [comment]
@rdp What's play it my way?

**Comment by rdp** [comment]
It's basically an online streaming editor to stream "amazon edited" as it
were: https://playitmyway.org/

**Comment by misbach** [comment]
Thanks! I had not heard of that before.

**Comment by rdp** [comment]
Yeah it's basically brand new and a work in progress, feedback welcome,
data entry helpers welcome :)
